ComputerVision/ImageNet/configs/resnet18/sparseift/sparsewide_static.yaml

```diff
@@ -0,0 +1,18 @@
+# Seed
```

Comment:
since we don't have results with this - let's not expose this config to the user.